index.js

```diff
@@ -16,6 +16,31 @@ function readUInt64LE(buf, offset = 0) {
 	return n;
 }
 
+const tarHeaderChecksumMatches = buf => { // Does not check if checksum field was valid
+	if (buf.length < 512) {
+		return false;
+	}
+
+	let sum = 256;
+	let signedBitCount = 0;
+
+	for (let i = 0; i < 148; i++) {
+		const byte = buf[i];
+		sum += byte;
+		signedBitCount += byte & 0x80;
```

Comment:
Can you move `0x80` to a variable with a descriptive name?

Reply:
But it's not a variable, it's bitmask.

Reply:
done